Library/Homebrew/cmd/install.rb

```diff
@@ -1,3 +1,51 @@
+#:* `install` [`--debug`] [`--env=`<std>|<super>] [`--ignore-dependencies`] [`--only-dependencies`] [`--cc=`<compiler>] [`--build-from-source`|`--force-bottle`] [`--devel`|`--HEAD`] <formula>:
```

Comment:
Yes, the rubocop for space after `#` doesn't complain for some special characters, see [leading_comment_space.rb](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/leading_comment_space.rb#L17).